lightning/src/onion_message/messenger.rs

```diff
@@ -130,26 +160,37 @@ pub enum SendError {
 	BufferFull,
 }
 
-impl<Signer: Sign, K: Deref, L: Deref> OnionMessenger<Signer, K, L>
+/// Handler for custom onion messages.
```

Comment:
Would be good to have a more descriptive explanation of how to implement this either here or maybe better in the `OnionMessenger` docs. Doesn't have to be very long so long as the whole picture is described somewhere.